specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2020-01-01-preview/managedcluster.json

```diff
@@ -0,0 +1,1080 @@
+            "description": "The detailed error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceFabric/managedClusters/{clusterName}": {
+      "get": {
+        "tags": [
+          "ManagedCluster"
+        ],
+        "operationId": "ManagedClusters_Get",
+        "summary": "Gets a Service Fabric managed cluster resource.",
+        "description": "Get a Service Fabric managed cluster resource created or in the process of being created in the specified resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "x-ms-examples": {
+          "Get a cluster": {
+            "$ref": "./examples/ManagedClusterGetOperation_example.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ManagedCluster"
+            }
+          },
+          "default": {
+            "description": "The detailed error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagedCluster"
+        ],
+        "operationId": "ManagedClusters_Create",
+        "summary": "Creates or updates a Service Fabric managed cluster resource.",
+        "description": "Create or update a Service Fabric managed cluster resource with the specified name.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The cluster resource.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedCluster"
+            },
+            "x-ms-parameter-location": "method"
+          }
+        ],
+        "x-ms-examples": {
+          "Put a cluster with minimum parameters": {
+            "$ref": "./examples/ManagedClusterPutOperation_example_min.json"
+          },
+          "Put a cluster with maximum parameters": {
+            "$ref": "./examples/ManagedClusterPutOperation_example_max.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
```

Comment:
>"x-ms-long-running-operation": true, [](start = 8, length = 36)

also add "x-ms-long-running-operation-options" extension with "final-state-via" property.
``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "location"
}
```

or

``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "azure-async-operation"
}
```

Reply:
same for all LRO operations (with 201 or 202)

---
In reply to: [368180555](https://github.com/Azure/azure-rest-api-specs/pull/8003#discussion_r368180555) [](ancestors = 368180555)